armi/utils/plotting.py

```diff
@@ -38,3 +53,724 @@ def colorGenerator(skippedColors=10):
         for i in range(start, len(colors), skippedColors):
             if colors[i] not in excludedColors:
                 yield colors[i]
+
+
+def plotBlockDepthMap(
+    core,
+    param="pdens",
+    fName=None,
+    bare=False,
+    cmapName="jet",
+    labels=(),
+    labelFmt="{0:.3f}",
+    legendMap=None,
+    fontSize=None,
+    minScale=None,
+    maxScale=None,
+    axisEqual=False,
+    makeColorBar=False,
+    cBarLabel="",
+    title="",
+    shuffleArrows=False,
+    titleSize=25,
+    depthIndex=0,
+):
+    """
+    Plot a param distribution in xy space with the ability to page through depth.
+
+    Notes
+    -----
+    This is useful for visualizing the spatial distribution of a param through the core.
+    Blocks could possibly not be in alignment between assemblies, but the depths
+    viewable are based on the first fuel assembly.
```

Comment:
It may be worth exposing this through a simple script somewhere as an entry point. If this gets powerful enough, it may even be worth creating a proper entry point for it. At the very least, a usage example would be nice. It's just a bit of a shame to have a tool like this buried so deep

Reply:
Yeah, I wrote a test for demonstration, but I was also thinking of adding it to the ipython tutorial instead of one of the get face map calls. I'll definitely be using it when I put together more C5G7 stuff.

Reply:
added to data_model.ipynb